src/api/app/policies/project_policy.rb

```diff
@@ -17,4 +17,8 @@ def update?
   def destroy?
     update?
   end
+
+  def new_incident?
```

Comment:
Nope! So you would prefer to remove the new_incident and use the update policy instead? Never thought about it, but sounds reasonable ;) I'll change it!